gtsam/geometry/Point3.cpp

```diff
@@ -77,6 +77,7 @@ double dot(const Point3 &p, const Point3 &q, OptionalJacobian<1, 3> H1,
 
 Point3Pair mean(const std::vector<Point3Pair> &abPointPairs) {
   const size_t n = abPointPairs.size();
+  if (n == 0) throw std::invalid_argument("input should have at least 1 pair of points");
```

Comment:
Slightly more descriptive error message?

`Point3Pair::mean input vector is empty`

Reply:
sure